README.md

```diff
@@ -241,7 +241,7 @@ Copyright &copy; 2020 [Mineiros GmbH][homepage]
 [badge-build]: https://github.com/mineiros-io/terraform-aws-iam-user/workflows/CI/CD%20Pipeline/badge.svg
 [badge-semver]: https://img.shields.io/github/v/tag/mineiros-io/terraform-aws-iam-user.svg?label=latest&sort=semver
 [badge-license]: https://img.shields.io/badge/license-Apache%202.0-brightgreen.svg
-[badge-terraform]: https://img.shields.io/badge/terraform-0.14,%200.13,%200.12.20+-623CE4.svg?logo=terraform
+[badge-terraform]: https://img.shields.io/badge/terraform-0.15,%200.13,%200.12.20+-623CE4.svg?logo=terraform
```

Comment:
0.14 shoudl be added back again

Reply:
I am not a big fan of listing every version - this will not scale and look pretty bad at some point. Also, this is a badge and should not block the PR from going back to master.

Reply:
so imho: we should not list every single version but simply use semver to express the range of versions.  It should be fine the way it is.

Reply:
the badge now says 0.15,0.13,0.12 ... missing 0.14 i do not see this should be this way.

```suggestion
[badge-terraform]: https://img.shields.io/badge/terraform-0.15%20|0.14%20|%200.13%20|%200.12.20+-623CE4.svg?logo=terraform
```

Reply:
resolved in 3e3d930

Reply:
not resolved yet

Reply:
resolved 140af4e